internal/aws/awsutil/conn.go

```diff
@@ -160,6 +163,38 @@ func GetAWSConfigSession(logger *zap.Logger, cn ConnAttr, cfg *AWSSessionSetting
 	return config, s, nil
 }
 
+// GetAWSConfig returns AWS config and session instances.
+func GetAWSConfig(logger *zap.Logger, settings *AWSSessionSettings) (awsv2.Config, error) {
+	var err error
```

Comment:
```suggestion
```
this is set line 170

Reply:
Yeah, thanks - too much copy+paste from the aws-sdk-go-v1 implementation - cleaned up!